pipelines/multi-branch/tracker.yml

```
@@ -0,0 +1,99 @@
+resource_types:
+- name: git-branches
+  type: registry-image
+  source:
+    repository: aoldershaw/git-branches-resource
+
+- name: terraform
+  type: registry-image
+  source:
+    repository: ljfranklin/terraform-resource
+
+resources:
+- name: feature-branches
+  type: git-branches
+  source:
+    uri: https://github.com/concourse/examples
+    branch_regex: 'feature/(?P<feature>.*)'
```

same question as other PR, bad regex?
```suggestion
    branch_regex: 'feature/.*'
```